arxiv/util/tests/test_authors.py

```diff
@@ -1,7 +1,7 @@
 """Tests for author and affiliation parsing."""
 from unittest import TestCase
 
-from ..authors import parse_author_affil, split_authors
```

Comment:
Relative imports are often used in tests so if the arxiv.util is moved, the tests don't need to change. This is probably not an issue with modern refactoring IDEs.

No change requested.